instructure/canvas-ios
# Support sso universal links

refs: MBL-14143
affects: Parent, Teacher
release note: none

Test plan
1. install [Student | Parent | Teacher] (choose 1 app)
2. choose the same app on web and display qr code to login
3. scan code , click to open in app
4. you should be logged in
5. repeat steps 1-4  w/ other 2 apps

Note: with multiple apps installed i had to delete some apps to prevent it from trying the same app again.
